[CI] MultiClusterSpecStatsFirstLastIT test {csv-spec:stats_first_last.Test aggregating on integers from row} failing

**Build Scans:**
- [elasticsearch-intake #44582 / 9.5.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/wxdmp22wmgjmw)
- [elasticsearch-pull-request #167061 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/3c22lbcyxs4ek)
- [elasticsearch-pull-request #167061 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/sb74wwgyavi66)
- [elasticsearch-pull-request #167045 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/bcopkl7g37szy)
- [elasticsearch-pull-request #167045 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/hguwbqgztbfn6)
- [elasticsearch-pull-request #167025 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/5meabxkgddd3o)
- [elasticsearch-pull-request #166890 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/hywp5ldcposi2)
- [elasticsearch-pull-request #166890 / 9.5.0_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/e2afpcsikyq7u)
- [elasticsearch-pull-request #166848 / 8.19.20_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/2g2nxujnnroks)
- [elasticsearch-pull-request #166848 / 9.3.9_part_3_bwc-snapshots](https://gradle-enterprise.elastic.co/s/omjzo5bb3ckgs)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:multi-clusters:v9.5.0#csvSpecTests#oldToNew" -Dtests.class="org.elasticsearch.xpack.esql.ccq.MultiClusterSpecStatsFirstLastIT" -Dtests.method="test {csv-spec:stats_first_last.Test aggregating on integers from row}" -Dtests.seed=705E62A7F22D5196 -Dtests.bwc=true -Dtests.locale=sc -Dtests.timezone=Mexico/BajaSur -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.ccq.MultiClusterSpecStatsFirstLastIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bcsv-spec%3Astats_first_last.Test%20aggregating%20on%20integers%20from%20row%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Context: {is_running=false, took=5, is_partial=false, completion_time_in_millis=1785168755092, documents_found=0, values_loaded=0, rows_emitted=31, bytes_read=0, read_nanos=0, cpu_nanos=1645026, start_time_in_millis=1785168755087, expiration_time_in_millis=1786032754925, columns=[{name=first_val, type=date}, {name=last_val, type=date}], values=[[2025-11-25T00:00:01.000Z, 2025-11-25T00:00:05.000Z]]}
Expected a list containing
0: expected "Line 15:10: evaluation of [TO_LONG(number)] failed, treating result as null. Only first 20 failures recorded." but was <missing>
1: expected "Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number []" but was <missing>
```

**Issue Reasons:**
- [main] 1 consecutive failures in step 9.5.0_bwc-snapshots
- [main] 2 consecutive failures in step 8.19.20_part_3_bwc-snapshots
- [main] 2 consecutive failures in step 9.3.9_part_3_bwc-snapshots
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 16 failures in test test {csv-spec:stats_first_last.Test aggregating on integers from row} (2.1% fail rate in 755 executions)
- [main] 2 failures in step 9.4.5_part_3_bwc-snapshots (1.6% fail rate in 125 executions)
- [main] 9 failures in step 9.5.0_part_3_bwc-snapshots (6.5% fail rate in 139 executions)
- [main] 2 failures in step 8.19.20_part_3_bwc-snapshots (100.0% fail rate in 2 executions)
- [main] 2 failures in step 9.3.9_part_3_bwc-snapshots (100.0% fail rate in 2 executions)
- [main] 5 failures in pipeline elasticsearch-pull-request (3.6% fail rate in 138 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-core-infra.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -544,9 +544,6 @@ tests:
 - class: org.elasticsearch.xpack.stateless.objectstore.gc.StaleTranslogsGCIT
   method: testIsolatedNodeDoesNotDeleteTranslogs
   issue: https://github.com/elastic/elasticsearch/issues/155130
-- class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecStatsFirstLastIT
-  method: test {csv-spec:stats_first_last.Test aggregating on integers from row}
-  issue: https://github.com/elastic/elasticsearch/issues/153700
 - class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecStatsIT
   method: test {csv-spec:stats.sumWithOverflowGroupingRow}
   issue: https://github.com/elastic/elasticsearch/issues/153434

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats_first_last.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats_first_last.csv-spec
@@ -49,8 +49,8 @@ ROW row = [
 | STATS first_val = FIRST(number, @timestamp)
 ;
 
-warning:Line 15:10: evaluation of [TO_LONG(number)] failed, treating result as null. Only first 20 failures recorded.
-warning:Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number []
+warningRegex:Line 15:10: evaluation of \[TO_LONG\(number\)\] failed, treating result as null. Only first 20 failures recorded.
+warningRegex:Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number \[\]
 
 first_val:long
 null
@@ -112,8 +112,8 @@ ROW row = [
 | STATS last_val = LAST(number, @timestamp) BY name
 ;
 
-warning:Line 15:10: evaluation of [TO_LONG(number)] failed, treating result as null. Only first 20 failures recorded.
-warning:Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number []
+warningRegex:Line 15:10: evaluation of \[TO_LONG\(number\)\] failed, treating result as null. Only first 20 failures recorded.
+warningRegex:Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number \[\]
 
 last_val:long | name:keyword
 4             | alpha
@@ -427,8 +427,8 @@ ROW row = [
         last_val = LAST(@timestamp, number)
 ;
 
-warning:Line 15:10: evaluation of [TO_LONG(number)] failed, treating result as null. Only first 20 failures recorded.
-warning:Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number []
+warningRegex:Line 15:10: evaluation of \[TO_LONG\(number\)\] failed, treating result as null. Only first 20 failures recorded.
+warningRegex:Line 15:10: org.elasticsearch.xpack.esql.core.InvalidArgumentException: Cannot parse number \[\]
 
 first_val:datetime       | last_val:datetime
 2025-11-25T00:00:01.000Z | 2025-11-25T00:00:05.000Z

